Example/Example/AppDelegate.swift

```diff
@@ -21,10 +21,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         locationManager.delegate = self
         self.requestLocationPermissions()
 
+        UserDefaults.standard.set("https://api-shicheng.radar-staging.com", forKey:"radar-host")
+        
+        Radar.stopTracking()
+        
         // Replace with a valid test publishable key
-        Radar.initialize(publishableKey: "prj_test_pk_0000000000000000000000000000000000000000")
+        Radar.initialize(publishableKey: "prj_test_pk_d9020be31f7cd2357d59a728008f469b1395ee4e")
```

Comment:
lets not commit this, also you kinda leaked your pub key. Let's not have this in the PR and you may want to rotate your staging key.